docs/production-deployment/cloud/nexus/using-nexus.mdx

```diff
@@ -0,0 +1,99 @@
+
+Unlike a traditional RPC, an arbitrary-duration Nexus Operation has an identity (ID) that lasts beyond the process lifetimes of the caller and handler, and can be used to re-attach to a long-lived Nexus Operation, for example one backed by a Temporal Workflow.
+
+When you schedule a Nexus Operation in a caller Workflow, a command is sent to Temporal to schedule the Operation, and then the Temporal Nexus Machinery is responsible for making the Nexus RPC calls on your behalf.
+This means you don’t have to use Nexus RPC directly, only the Temporal SDK along with a Temporal Service.
+
+![NexusYourCloud](/img/nexus/NexusYourCloud.png)
+
+Similar to executing an Activity, to execute a Nexus Operation from a Workflow:
+
+1. Caller Workflow will schedule a Nexus Operation using the Temporal SDK, which sends a `ScheduleNexusOperation` command in the existing `RespondWorkflowTaskCompleted` RPC which is recorded as `NexusOperationScheduled` in the caller’s Workflow history.
+2. Nexus handler Worker uses the Temporal SDK to `PollNexusTaskQueue` and process the Nexus request.
+3. Nexus handler Worker sends a `RespondNexusTaskCompleted` through the SDK, with the result of the Nexus request, which may be one of: Started, Completed, Failed, TimedOut, or Canceled. See [Nexus Operation Events](/references/events) for more info
+4. Caller Workflow then processes the Nexus Operation event through `PollWorkflowTaskQueue`.
+
+See the [Temporal Nexus](/nexus) encyclopedia entry for more details. 
+
+## Access Control
+
+Like Namespaces, a Nexus Endpoint is an account-scoped resource that is global within a Temporal Cloud account. 
+Any Developer role (or higher) in an account, who is also a Namespace Admin on the endpoint’s target Namespace, can manage (create, update, or delete) a Nexus Endpoint. 
+All users with a Read-only role (or higher) in an account, can view and browse the full list of Endpoints.
+
+Runtime access from a Workflow in a caller Namespace to a Nexus Endpoint is controlled by an allowlist policy (of caller Namespaces) for each Endpoint in the Nexus API registry.
+Workers authenticate with Temporal Cloud as they do today with mTLS certificates or API keys as allowed by the Namespace configuration.
+In Temporal Cloud, Nexus requests are sent from the caller’s Namespace to the handler’s Namespace over a secure multi-region mTLS Envoy mesh.
+
+See [Nexus Security](/nexus/security) for additional details.
+
+## Automatic Retries
+
+Once the caller Workflow schedules an Operation with the caller’s Temporal Service, the caller’s Nexus Machinery keeps trying to start the Operation, with automatic retries and exponential backoff. 
+If a Nexus Operation returns a retryable error when attempting to start, the Operation it will be retried up to the default Retry Policy’s maximum attempts and expiration interval.
+
+See [Nexus Automatic Retries](/nexus/services-operations#automatic-retries) for additional details.
+
+## Failure Handling
```

Comment:
do we mention anything about our own circuit breaking/throttling behavior in the docs?